vault/cluster/inmem_layer.go

```diff
@@ -114,12 +122,24 @@ func (l *InmemLayer) Dial(addr string, timeout time.Duration, tlsConfig *tls.Con
 		panic(fmt.Sprintf("%q attempted to dial itself", l.addr))
 	}
 
+	if l.forceTimeout == addr {
+		l.logger.Debug("forcing timeout", "addr", addr, "me", l.addr)
+		time.Sleep(time.Second * 20)
```

Comment:
Worth adding a comment explaining what this is meant to simulate?  It's clear to us now but a year from now this might look weird.

Reply:
Added!